content/en/docs/tasks/administer-cluster/kubelet-in-userns.md

```diff
@@ -34,14 +34,21 @@ If you are just looking for how to run a pod as a non-root user, see [SecurityCo
 
 ## Running Kubernetes inside Rootless Docker/Podman
 
-[kind](https://kind.sigs.k8s.io/) supports running Kubernetes inside a Rootless Docker or Rootless Podman.
+### kind
+
+[kind](https://kind.sigs.k8s.io/) supports running Kubernetes inside Rootless Docker or Rootless Podman.
 
 See [Running kind with Rootless Docker](https://kind.sigs.k8s.io/docs/user/rootless/).
 
-<!--
-[minikube](https://minikube.sigs.k8s.io/docs/) also plans to support Rootless Docker/Podman drivers.
-See [minikube issue #10836](https://github.com/kubernetes/minikube/issues/10836) to track the progress.
--->
+### minikube
+
+[minikube](https://minikube.sigs.k8s.io/) also supports running Kubernetes inside Rootless Docker.
+
+See [minkube Documentation/Drivers/docker](https://minikube.sigs.k8s.io/docs/drivers/docker/).
+
+Rootless Podman is not supported yet.
+
+<!-- Supporting rootless podman is discussed in https://github.com/kubernetes/minikube/issues/8719 -->
```

Comment:
We [avoid statements about the future](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-statements-about-the-future), so I'd either omit the “yet” or leave out the whole paragraph.

Reply:
removed "yet"